docs/configuration-options.md

```diff
@@ -0,0 +1,103 @@
+# Configuration Options
+
+You can pass an options object when you call `ShareButton` on an element to make things a little more flexible.
+
+Passing configuration options:
+
+```js
+config = {
+    networks: {
+        facebook: {
+            app_id: '12345'
+        }
+    }
+}
+
+var share = new ShareButton('.share-button', config);
+```
+
+All options:
+```js
```

Comment:
it actually might be easier to just describe these options with JSdoc... then you can generate some nice API docs from it